### DISCUSSION TITLE
How do I remove the wayback machine id in the address bar?

### DISCUSSION THREAD

**Discussion by Marmelucos** [atauenis/webone#129] [Q&A] [answered]
is there any way to hide the id of the wayback machine in the address bar, leaving only that of the visited site?

I'm using the following command to create the redirect to the wayback machine: 

[Edit:^http://www.cade.com.br/]
AddRedirect=http://web.archive.org/web/2000id_/%URL%

big thanks!

![image](https://github.com/atauenis/webone/assets/67889442/d546bd02-8331-455b-8abb-0e8ee661baec)

**Comment by atauenis** [comment, marked as answer]
Use
```INI
[Edit:^http://www.cade.com.br/]
AddInternalRedirect=http://web.archive.org/web/20001027204403id_/%URL%
```
If `20001027204403` is not the needed date, specify some other date which is exists in Wayback Machine archive. And some sites don't work as intended with this, due to archive limitations.

**Reply by Marmelucos** [reply]
How do I make the wayback bring up a page from a specific randomized year, Is that possible?

**Reply by atauenis** [reply]
At this moment - not possible. The wayback machine in case if the date is not archived, makes a redirect to page with saved date, making the URL starting with `http://web.archive.org/web/`.
